### ISSUE TITLE
[JSON] RSS uses tuple for ordering

### ISSUE THREAD

**Issue by chaen** [DIRACGrid/DIRAC#4321]
To be checked if it can be converted to a list

```
============================================= Encoding tuples =============================================
  File "/scratch/workspace/jsonSerialization/ServerInstallDIR/DIRAC/Core/scripts/dirac-agent.py", line 45, in <module>
    agentReactor.go()

  File "/scratch/workspace/jsonSerialization/ServerInstallDIR/DIRAC/Core/Base/AgentReactor.py", line 146, in go
    timeToNext = self.__scheduler.executeNextTask()

  File "/scratch/workspace/jsonSerialization/ServerInstallDIR/DIRAC/Core/Utilities/ThreadScheduler.py", line 134, in executeNextTask
    self.__executeTask( taskId )

  File "/scratch/workspace/jsonSerialization/ServerInstallDIR/DIRAC/Core/Utilities/ThreadScheduler.py", line 187, in __executeTask
    task[ 'func' ]( *task[ 'args' ] )

  File "/scratch/workspace/jsonSerialization/ServerInstallDIR/DIRAC/Core/Base/AgentModule.py", line 365, in am_go
    cycleResult = self.__executeModuleCycle()

  File "/scratch/workspace/jsonSerialization/ServerInstallDIR/DIRAC/Core/Base/AgentModule.py", line 430, in __executeModuleCycle
    result = self.am_secureCall(executors[0][0], executors[0][1])

  File "/scratch/workspace/jsonSerialization/ServerInstallDIR/DIRAC/Core/Base/AgentModule.py", line 327, in am_secureCall
    result = functor(*args)

  File "/scratch/workspace/jsonSerialization/ServerInstallDIR/DIRAC/ResourceStatusSystem/Agent/TokenAgent.py", line 74, in execute
    interestingTokens = self._getInterestingTokens(element)

  File "/scratch/workspace/jsonSerialization/ServerInstallDIR/DIRAC/ResourceStatusSystem/Agent/TokenAgent.py", line 100, in _getInterestingTokens
    meta={'older': ('TokenExpiration', tokenExpLimit)})

  File "/scratch/workspace/jsonSerialization/ServerInstallDIR/DIRAC/ResourceStatusSystem/Client/ResourceStatusClient.py", line 128, in selectStatusElement
    return self._getRPC().select(element + tableType, self._prepare(locals()))

  File "/scratch/workspace/jsonSerialization/ServerInstallDIR/DIRAC/Core/DISET/RPCClient.py", line 43, in __call__
    return self.__doRPCFunc( self.__remoteFuncName, args )

  File "/scratch/workspace/jsonSerialization/ServerInstallDIR/DIRAC/Core/DISET/RPCClient.py", line 90, in __doRPC
    return self.__innerRPCClient.executeRPC( sFunctionName, args )

  File "/scratch/workspace/jsonSerialization/ServerInstallDIR/DIRAC/Core/DISET/private/InnerRPCClient.py", line 64, in executeRPC
    retVal = transport.sendData( S_OK( list(args) ) )

  File "/scratch/workspace/jsonSerialization/ServerInstallDIR/DIRAC/Core/DISET/private/Transports/BaseTransport.py", line 162, in sendData
    sCodedData = DEncode.encode(uData)

  File "/scratch/workspace/jsonSerialization/ServerInstallDIR/DIRAC/Core/Utilities/DEncode.py", line 393, in encode
    g_dEncodeFunctions[type(uObject)](uObject, eList)

  File "/scratch/workspace/jsonSerialization/ServerInstallDIR/DIRAC/Core/Utilities/DEncode.py", line 361, in encodeDict
    g_dEncodeFunctions[type(dValue[key])](dValue[key], eList)

  File "/scratch/workspace/jsonSerialization/ServerInstallDIR/DIRAC/Core/Utilities/DEncode.py", line 305, in encodeList
    g_dEncodeFunctions[type(uObject)](uObject, eList)

  File "/scratch/workspace/jsonSerialization/ServerInstallDIR/DIRAC/Core/Utilities/DEncode.py", line 361, in encodeDict
    g_dEncodeFunctions[type(dValue[key])](dValue[key], eList)

  File "/scratch/workspace/jsonSerialization/ServerInstallDIR/DIRAC/Core/Utilities/DEncode.py", line 361, in encodeDict
    g_dEncodeFunctions[type(dValue[key])](dValue[key], eList)

  File "/scratch/workspace/jsonSerialization/ServerInstallDIR/DIRAC/Core/Utilities/DEncode.py", line 328, in encodeTuple
    printDebugCallstack('Encoding tuples')

Calling frame: ('/scratch/workspace/jsonSerialization/ServerInstallDIR/DIRAC/Core/DISET/private/Transports/BaseTransport.py', 162)
With arguments  {'uObject': {'OK': True,
             'Value': ['SiteStatus',
                       {'Meta': {'older': ('TokenExpiration',
                                           datetime.datetime(2019, 10, 25, 1, 38, 12, 508867))}}]}}
====================================================================================================

```

**Comment by chaen** [comment]
In https://github.com/DIRACGrid/DIRAC/pull/4366

**Comment by chrisburr** [comment]
I think this can be closed now #4410 has been merged?
